src/main/webapp/script.js

```diff
@@ -44,6 +44,7 @@ function getZipCode() {
 
 //Array of the (currently 6 for this demo build) 15 listings gathered from the fetch request
 let listingsArray = [];
+let photoReferencesArray = [];
```

Comment:
Instead of two parallel arrays, could this have been done with one. That way each element in the listingsArray would 'have' a photoreference?

Reply:
Done